src/pylyrics/download_data.py

```diff
@@ -29,6 +29,15 @@ def download_data(dataset, file_path, columns):
     download_data("geomack/spotifyclassification", "data/spotify_attributes", ("song_title", "artist"))
     spotify_df = download_data("geomack/spotifyclassification", "data/spotify_attributes", ("song_title", "artist"))
     """
+    if not (type(dataset)) == str:
+        raise TypeError("Dataset should be of type string.")
+    if not (type(file_path)) == str:
+        raise TypeError("File_path should be of type string.")
+    if not(type(columns)) == list:
+        raise TypeError("The column names should be of type list")
+    if not(len(columns)) == 2:
+        raise TypeError("Two columns should be retrieved")
+
```

Comment:
Instead of use try-except, can do sth like 
``` 
 directory = os.path.dirname(file_path)
        if not os.path.exists(directory):
            os.makedirs(directory)
```

Reply:
Yes, made the suggested changes and tested.